packages/api/src/command/medical/cq-directory/search-cq-directory.ts

```diff
@@ -35,16 +38,22 @@ export async function searchNearbyCQOrganizations({
   radiusInMiles?: number;
 }): Promise<CQOrgBasicDetails[]> {
   const radiusInMeters = convert(radiusInMiles).from("mi").to("m");
-
   const patient = await getPatientOrFail({ id: patientId, cxId });
-  const coordinates = await geocodeAddresses(patient.data.address);
+  const addresses = await upsertGeographicCoordinates({
```

Comment:
Can't we store the coordinates while we query the directory?

Reply:
this is for the patient's address 

the cq directory provides coordinates for the organizations and we are storing those

Reply:
I'm not super sure about the context of my comment (it was not very clear), but I think what it means is that we're calculating and storing the patient address' coordinates upon Patient create and update, so why would we want to calculate this when we query?

Querying is not the time to calculate and make updates, right?

Also, IFF we keep updating the address, I think we should be specific on the function name, indicating it updates the patient's address - one wouldn't expect that from a pure "search" function

Reply:
Yup. This also lines up what I'd said about my own work here: https://github.com/metriport/metriport/pull/1323#discussion_r1423308122

Fixed now 👍